BuildResidentialHPXML/measure.rb

```diff
@@ -2635,6 +2636,306 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('electric_panel_load_kitchen_fans_addition', false)
+    arg.setDisplayName('Electric Panel: Kitchen Fans Addition')
+    arg.setDescription("Whether the kitchen fans is a new panel load addition to an existing service panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_bathroom_fans_power', false)
+    arg.setDisplayName('Electric Panel: Bathroom Fans Power')
+    arg.setDescription("Specifies the panel load bathroom fans power. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
+    arg.setUnits('W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('electric_panel_load_bathroom_fans_addition', false)
+    arg.setDisplayName('Electric Panel: Bathroom Fans Addition')
+    arg.setDescription("Whether the bathroom fans is a new panel load addition to an existing service panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_water_heater_power', false)
+    arg.setDisplayName('Electric Panel: Water Heater Power')
+    arg.setDescription("Specifies the panel load water heater power. Only applies to electric water heater. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
+    arg.setUnits('W')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeChoiceArgument('electric_panel_load_water_heater_voltage', electric_panel_voltage_choices, false)
+    arg.setDisplayName('Electric Panel: Water Heater Voltage')
+    arg.setDescription("Specifies the panel load water heater voltage. Only applies to #{HPXML::WaterHeaterTypeHeatPump} (compressor). If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
+    arg.setUnits('V')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeBoolArgument('electric_panel_load_water_heater_addition', false)
+    arg.setDisplayName('Electric Panel: Water Heater Addition')
+    arg.setDescription("Whether the water heater is a new panel load addition to an existing service panel. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeDoubleArgument('electric_panel_load_clothes_dryer_power', false)
+    arg.setDisplayName('Electric Panel: Clothes Dryer Power')
+    arg.setDescription("Specifies the panel load power. Only applies to electric clothes dryer. If not provided, the OS-HPXML default (see <a href='#{docs_base_url}#panel-loads'>Panel Loads</a>) is used.")
```

Comment:
For inputs that only apply to electric appliances, add "electric" to the arguments/names?

Reply:
https://github.com/NREL/OpenStudio-HPXML/pull/1946/commits/64cdffdcf61f3ab9f4416252b91e4d6366558d41